src/test/java/lotto/step2/WinningStatisticsTest.java

```diff
@@ -17,23 +17,24 @@
 
     @ParameterizedTest(name = "profitsTest")
     @MethodSource("arrayProvider")
-    public void profitsTest(Lottos inputLottos, Lotto winningLotto, double expected) {
-        WinningStatistics statistics = new WinningStatistics(inputLottos, winningLotto);
+    public void profitsTest(Lottos inputLottos, WinningNumbers winningNumbers, double expected) {
+        WinningStatistics statistics = new WinningStatistics(inputLottos, winningNumbers);
         assertThat(statistics.profits()).isEqualTo(expected);
     }
 
     static Stream<Arguments> arrayProvider() {
 
         return Stream.of(
-                Arguments.of((Object)new Lottos(new ArrayList<Lotto>() {
-                                {
-                                        add(new Lotto(new String[]{"1","2","3","4","5","7"}));
-                                        add(new Lotto(new String[]{"1","2","3","4","7","8"}));
-                                        add(new Lotto(new String[]{"1","2","3","4","7","8"}));
-                                }
+                Arguments.of((Object) new Lottos(new ArrayList<Lotto>() {
+                            {
+                                add(new Lotto(new String[]{"1", "2", "3", "4", "5", "7"}));
+                                add(new Lotto(new String[]{"1", "2", "3", "4", "7", "8"}));
+                                add(new Lotto(new String[]{"1", "2", "3", "4", "7", "8"}));
+                                add(new Lotto(new String[]{"1", "2", "3", "4", "5", "45"}));
                             }
+                        }
                         )
-                        ,(Object) new Lotto(new String[]{"1","2","3","4","5","6"}), 533.33)
+                        , (Object) new WinningNumbers(new String[]{"1", "2", "3", "4", "5", "6"}, 45), 7900)
```

Comment:
```suggestion
                        , new WinningNumbers(new String[]{"1", "2", "3", "4", "5", "6"}, 45), 7900)
```

Object 로의 타입 캐스팅 없이도, 테스트를 할 수 있도록 수정해주세요.
Object 로의 타입 캐스팅 뿐만 아니라 Object 타입의 사용 자체를 가능한 허용하지 않으려합니다.
이유는 정적 타입언어로서의 자바의 장점이 사라지기 때문입니다.
Object 타입으로 퉁치면 컴파일 타임에서든, 런타임에서든 비효율성과 버그 가능성이 발생합니다.

Reply:
[링크](https://stackoverflow.com/questions/66675658/junit-test-parameterizedtests-no-implicit-conversion-of-type-java-lang-inte)를 참조하면 Argument.of의 argument가 1개이고 배열일 때는 혼동이 있어서 Object로의 캐스팅이 필요한 거 같습니다

Reply:
그런 경우라면 차라리 Object 가 아니라, WinningNumbers 처럼 구체적인 객체 그 자체로 타입 캐스팅하면 어떨까요?
Object 로 타입 캐스팅하는 건 저는 꺼려지네요. (물론 이건 취향차이니깐, 꼭 따르지 않으셔도 됩니다.)